.eslintrc.json

```diff
@@ -14,6 +14,9 @@
       "class-methods-use-this": ["off"],
       "block-scoped-var": ["off"],
       "no-var": ["off"],
+      "no-redeclare": ["off"],
```

Comment:
The idea to use linters is to analyze source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Kindly do not modify linters configuration provided by Microverse. Use the files provided[ in this link](https://github.com/microverseinc/linters-config), please.